ci/fih_test_docker/fi_tester_gdb.sh

```diff
@@ -0,0 +1,200 @@
+#!/bin/bash
+
+# Copyright (c) 2020 Arm Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function skip_instruction {
```

Comment:
The body of this function should be indented (apart from the multi-line echo that is...).

Reply:
Ok. Although the `EOF` added for the next comment cannot be indented, so I left the whole gdb script part unindented.